index.html

```diff
@@ -1,81 +1,81 @@
 <!DOCTYPE html>
-<html lang = en>
-<head>
-	<meta charset="utf-8">
-	<title>Rocket Science</title>
-</head>
-<body>
-	<div id = "branding">
-		<img src = "" alt = "logo"/>
-		<h1>Rocket Science</h1>
-		<h2>Software Craftsmanship</h2>
-	</div>
+<html lang=en>
+  <head>
+    <meta charset="utf-8">
+    <title>Rocket Science</title>
+  </head>
+  <body>
+    <div id="branding">
+      <img src="" alt="logo"/>
+      <h1>Rocket Science</h1>
+      <h2>Software Craftsmanship</h2>
+    </div>
 
-	<div id = "introText">
-		<p>Lorem ipsum dolor sit amet, quo eu sanctus omnesque, an quas splendide pro, maiorum fabellas per et. Vix et prima bonorum adipiscing, ius zril inermis ex. Sea ne omnis modus tincidunt, eos an cetero singulis torquatos. Et erat pertinacia cum, instructior reprehendunt vel ut. Ex eligendi insolens sapientem per, duo id tale sumo propriae, ut falli inermis cum.</p>
-	</div>
+    <div id="introText">
+      <p>Lorem ipsum dolor sit amet, quo eu sanctus omnesque, an quas splendide pro, maiorum fabellas per et. Vix et prima bonorum adipiscing, ius zril inermis ex. Sea ne omnis modus tincidunt, eos an cetero singulis torquatos. Et erat pertinacia cum, instructior reprehendunt vel ut. Ex eligendi insolens sapientem per, duo id tale sumo propriae, ut falli inermis cum.</p>
+    </div>
 
-	<hr>
+    <hr>
 
-	<div id = "mainContent">
-		<h3>The Team</h3>
+    <div id="mainContent">
+      <h3>The Team</h3>
 
-		<div class = "teamMember">
-			<img src = "" alt = "headshot"/>
-			<h3>Name</h3>
-			<h4>Job title description</h4>
-			<p>Lorem ipsum dolor sit amet, quo eu sanctus omnesque, an quas splendide pro, maiorum fabellas per et. Vix et prima bonorum adipiscing, ius zril inermis ex. Sea ne omnis modus tincidunt, eos an cetero singulis torquatos. Et erat pertinacia cum, instructior reprehendunt vel ut.</p>
-			<img src = "" alt = "email"/> <img src = "" alt = "link"/> <img src = "" alt = "linkedIn"/>
-		</div>
+      <div class="teamMember">
+        <img src="" alt="headshot"/>
+        <h3>Name</h3>
+        <h4>Job title description</h4>
+        <p>Lorem ipsum dolor sit amet, quo eu sanctus omnesque, an quas splendide pro, maiorum fabellas per et. Vix et prima bonorum adipiscing, ius zril inermis ex. Sea ne omnis modus tincidunt, eos an cetero singulis torquatos. Et erat pertinacia cum, instructior reprehendunt vel ut.</p>
+        <img src="" alt="email"/> <img src="" alt="link"/> <img src="" alt="linkedIn"/>
```

Comment:
alt text for all of these should be in title case; treat it as though it was a heading on part of the document. `Email` and `LinkedIn`!